### PR TITLE
plat-ti: move some CFG_'s from platform_config.h to conf.mk

### PR DESCRIPTION
**Pull request by etienne-lms**
@glneo,
This P-R is small changes. It removes `CFG_` definitions from **plat-ti/platform_config.h** file.
Could you please ack the changes or report issues or comments you would like to share?
regards

### CODE REVIEW

File: core/arch/arm/plat-ti/conf.mk [hunk 1]
```diff
@@ -7,27 +7,38 @@ CFG_SECURE_DATA_PATH ?= y
 
 ifeq ($(CFG_SECURE_DATA_PATH),y)
 CFG_TEE_SDP_MEM_SIZE ?= 0x00400000
-else
-CFG_TEE_SDP_MEM_SIZE ?= 0x0
```
Comment (glneo) [inline core/arch/arm/plat-ti/conf.mk]: This change seems unrelated to the rest.

Does this break the CFG_SECURE_DATA_PATH == y case to have CFG_TEE_SDP_MEM_SIZE undefined like this?

Comment (etienne-lms) [reply core/arch/arm/plat-ti/conf.mk]: True. That change is not required. `CFG_TEE_SDP_MEM_SIZE` is not used even if defined when SDP is disable
Tell me if you prefer I restore to removed code or not.

**Comment by glneo** [comment]
Everything else looks good:

Acked-by: Andrew F. Davis <afd@ti.com>

**Comment by etienne-lms** [comment]
Thanks Andrew.

For info, in a later p-r, i will propose a change to get the internal/external secure rams and shm locations from generic config directives. I'll notify you by them with a update proposal for **plat-ti**. You'll see what it is about and feel free to reject proposed change to keep source file hardcoded locations.

**Comment by etienne-lms** [comment]
squashed, tag applied.

**Comment by jforissier** [comment]
CI complains about the `Acked-by:` tag.

**Comment by glneo** [comment]
Hmm, looks like github ate my email, lets try again:

```
Acked-by: Andrew F. Davis <afd@ti.com>
```

**Comment by etienne-lms** [comment]
thanks

**Comment by jforissier** [comment]
All good, thanks guys.
